src/templates/sermons.js

```diff
@@ -2,16 +2,51 @@ import React from "react"
 import Layout from "../components/layout"
 import './sermons.scss'
 import Pagination from "../components/molecules/pagination"
+import {graphql} from "gatsby";
+import moment from "moment";
 
-export default function Sermons({ pageContext }) {
-    const sermons = pageContext.nodes,
+export default function Sermons({pageContext, data}) {
+    if (!data) {
+        return <p>Page query failed</p>
+    }
+
+    const sermons = data.avorg.sermons.nodes,
         {total, current} = pageContext.pagination;
 
     return (
         <Layout className={'template-sermons'}>
-            <ul>{sermons.map((n, i) => <li>{n.title}</li>)}</ul>
+            <table>
+                {sermons.map((n, i) => <tr>
+                    <td><img src={n.imageWithFallback.url} alt={n.title} /></td>
+                    <td><a href={`/en/sermons/${n.id}`}>{n.title}</a></td>
```

Comment:
I think you could use `Recording.canonicalPath` here if you wanted.

Reply:
Isn't the API still using legacy paths?

Reply:
Yeah, it is. I would recommend if you want to use new paths now extract the url path generation out to a method so we can at least find all the places we'll need to change to `canonicalPath` when the time comes.